packages/babel-plugin-react-jsx/src/TransformJSXToReactBabelPlugin.js

```diff
@@ -24,6 +24,47 @@
 'use strict';
 
 const esutils = require('esutils');
+const {
+  isModule,
+  addNamespace,
+  addNamed,
+  addDefault,
+} = require('@babel/helper-module-imports');
+
+// These are all the valid auto import types (under the config autoImport)
+// that a user can specific
+const IMPORT_TYPES = {
+  none: 'none', // default option. Will not import anything
+  require: 'require', // var _react = require("react");
+  namespace: 'namespace', // import * as _react from "react";
+  default: 'default', // import _default from "react";
```

Comment:
I think we should probably rename this to `defaultExport` to avoid confusion since this isn't the "default" (as in "unless specified") mode.